source/system/Exception.ooc

```diff
@@ -200,9 +200,6 @@ OutOfBoundsException: class extends Exception {
 	init: func ~noOrigin (accessOffset, elementLength: Int) {
 		message = "Trying to access an element at offset %i, but size is %i!" format(accessOffset, elementLength)
 	}
-	kean_exception_outOfBoundsException_throw: static unmangled func (accessOffset, elementLength: Int) {
```

Comment:
yes :) it was mistakenly removed, re-added it again.

@marcusnaslund @sebastianbaginski @emilwestergren